src/maps/pallets.json

```diff
@@ -183,7 +171,31 @@
     "defaultPallet": "XTokens",
     "supportedPallets": ["PolkadotXcm", "XTokens"]
   },
-  "Efinity": {
+  "Manta": {
+    "defaultPallet": "XTokens",
+    "supportedPallets": ["PolkadotXcm", "XTokens"]
+  },
+  "Genshiro": {
+    "defaultPallet": "PolkadotXcm",
+    "supportedPallets": ["PolkadotXcm"]
+  },
+  "Nodle": {
+    "defaultPallet": "XTokens",
+    "supportedPallets": ["PolkadotXcm", "XTokens"]
+  },
+  "OriginTrail": {
+    "defaultPallet": "PolkadotXcm",
+    "supportedPallets": ["PolkadotXcm"]
+  },
+  "Pendulum": {
+    "defaultPallet": "XTokens",
+    "supportedPallets": ["PolkadotXcm", "XTokens"]
+  },
+  "Zeidgeist": {
```

Comment:
You have typo here sir @michaeldev5

Reply:
Thanks for pointing this out, we will fix it asap.

Reply:
Fixed in #69